nuxeo-liveconnect-dropbox-core/src/main/java/org/nuxeo/ecm/liveconnect/dropbox/DropboxBlobProvider.java

```diff
@@ -105,16 +106,17 @@ public InputStream getThumbnail(ManagedBlob blob) throws IOException {
         LiveConnectFileInfo fileInfo = toFileInfo(blob);
         String filePath = fileInfo.getFileId();
         try {
-            DbxClient.Downloader downloader = getDropboxClient(getCredential(fileInfo)).startGetThumbnail(
-                    DbxThumbnailSize.w64h64, DbxThumbnailFormat.bestForFileName(filePath, DbxThumbnailFormat.JPEG),
-                    filePath, null);
-
+            DbxDownloader downloader = getDropboxClient(getCredential(fileInfo)).files()
+                    .getThumbnailBuilder(filePath)
+                    .withFormat(ThumbnailFormat.JPEG)
+                    .withSize(ThumbnailSize.W64H64)
+                    .start();
             if (downloader == null) {
                 return null;
             }
-            return downloader.body;
+            return downloader.getInputStream();
         } catch (DbxException e) {
-            throw new IOException("Failed to get Dropbox file thumbnail " + e);
+          return null;
```

Comment:
Some indent issue here.
Also please log the exception at least as WARN so that we can diagnose what's happening and improve the code.